python/langsmith/evaluation/evaluator.py

```diff
@@ -101,6 +102,9 @@ async def aevaluate_run(
         )
 
 
+_RUNNABLE_OUTPUT = Union[EvaluationResult, EvaluationResults, dict]
+
+
 class DynamicRunEvaluator(RunEvaluator):
```

Comment:
not related to PR but docstring suggests we need to use `@run_evaluator` decorator but don't think that's true anymore?

Reply:
You could initialize it directly or via the run_evaluator docstring.

We wrap user functions using this within the `evaluate()` function to promote them to `RunEvaluator`'s